src/mixin.ts

```diff
@@ -83,7 +82,7 @@ export function mixin(Vue: VueConstructor) {
     const ctx = createSetupContext(vm)
 
     // fake reactive for `toRefs(props)`
-    def(props, PropsReactive, true)
+    def(props, '__ob__', {})
```

Comment:
Or should it be `def(props, '__ob__', (observe({}) as any).__ob__)`?

Reply:
using the observable should be "safer", might cause some unexpected side effects, but both would cause that, I would say go for observable for now and see how it goes